envs/external-airflow-db/.whirl.env

```diff
@@ -1,13 +1,14 @@
 # postgres env vars
 POSTGRES_HOST=postgresdb
 POSTGRES_PORT=5432
-POSTGRES_PASSWORD=p@ssw0rd
+POSTGRES_PASSWORD=pAssw0rd
```

Comment:
Haha, was time to rotate that one ;-)

Reply:
Ja en de extra `@` in de jdbcurl vorm zorgde ervoor dat de db connectie check niet goed ging.